demos/common/monitors/src/cpu_monitor.cpp

```diff
@@ -2,7 +2,8 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include "cpu_monitor.h"
+#include <monitors/cpu_monitor.h>
```

Comment:
Here, in memory_monitor.cpp, presenter.cpp and args_helper.cpp in similar places quotes should be used instead of brackets . cpu_monitor.h is a header that corresponds to the same library so it is "local" header for that .cpp file.

Reply:
Seems like an arbitrary distinction to me, but OK.

Reply:
There is args_helper.cpp left.

Reply:
Oh, right. Fixed.